lib/acme_server/jobs.ex

```diff
@@ -1,8 +1,18 @@
 defmodule AcmeServer.Jobs do
+  @moduledoc false
+
   def start_link() do
+    # This is the supervision subtree where concurrent jobs of AcmeServer are running.
+    # Currently, the only type of job is an http verifier, which issues a challenge
+    # request to the site which wants to be certified.
+
     Supervisor.start_link(
       [
+        # Registry is used to register a job under an arbitrary id. This is
```

Comment:
“Under and arbitrary id”.  I think it’s under a job based on the site to avoid duplicates of the same job.  Consider rewording.